### PR TITLE
Add a snap store icon for DCGM

### PR DESCRIPTION
**Pull request by aieri**
This commit adds an icon for the snap. The icon is pulled from the
official logo[0] with the transparent background trimmed to 256x256px as
recommended in the snapcraft documentation[1]

[0] https://assets.nvidiagrid.net/ngc/logos/DCGM.png
[1] https://snapcraft.io/docs/snapcraft-yaml-schema#icon

### CODE REVIEW

**Review by Deezzir — CHANGES_REQUESTED** [review]
Blocked while waiting for NVIDIA's explicit approval

**Review by gabrielcocenza — APPROVED** [review]

**Comment by aieri** [comment]
This is how the store will look after this change

image


image

**Comment by aieri** [comment]
We have received approval from NVIDIA via email from Edrick Wong. We can go ahead and merge the icon.

**Review by Deezzir — APPROVED** [review]

**Pull request merged**
